modules/nf-core/regtools/junctionsextract/environment.yml

```diff
@@ -0,0 +1,7 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "regtools_junctionsextract"
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - "bioconda::regtools=0.5.0"
```

Comment:
Looks like this version is 6 years old 🫠 https://prefix.dev/channels/bioconda/packages/regtools

Not sure if that's intentional or just the most up-to-date biocontainers version.

Reply:
it was just the most recent biocontainers version and i don't specifically need a newer version

Reply:
We would like this version because it is working well, and we didn't test the other. We can upgrade the version when is in there so we can have both versions in modules.